event_files/topdown.json

```diff
@@ -1,18 +1,18 @@
 [
     {
-        "EventName": "TOPDON.SLOTS",
+        "EventName": "TOPDOWN.SLOTS",
         "EventCode": "0x0",
         "BriefDescription": "TMA slots available for an unhalted hart",
         "PublicDescription": "TMA slots available for an unhalted hart"
     },
     {
-        "EventName": "TOPDON.FRONTEND_BOUND",
+        "EventName": "TOPDOWN.FRONTEND_BOUND",
         "EventCode": "0x0",
         "BriefDescription": "TMA slots unused due to the frontend did not supply enough operations",
         "PublicDescription": "TMA slots unused due to the frontend did not supply enough operations"
     },
     {
-        "EventName": "TOPDON.BACEND_BOUND",
+        "EventName": "TOPDOWN.BACKEND_BOUND",
         "EventCode": "0x0",
         "BriefDescription": "TMA slots unused due to the due to lack of backend resources",
         "PublicDescription": "TMA slots unused due to the due to lack of backend resources"
```

Comment:
Based on the Intel TMA paper, shouldn't there be a fourth category for level 1, i.e.  TOPDOWN.BADSPECULATION?

https://www.researchgate.net/publication/269302126_A_Top-Down_method_for_performance_analysis_and_counters_architecture